gp-templates/translations.php

```diff
@@ -217,12 +217,12 @@
 						'filters[status]', // TODO: show only these, which user is allowed to see afterwards.
 						array(
 							'current_or_waiting_or_fuzzy_or_untranslated' => __( 'Current/waiting/fuzzy + untranslated (All)', 'glotpress' ),
-							'current'                                     => __( 'Current only', 'glotpress' ),
-							'old'                                         => __( 'Approved, but obsoleted by another translation', 'glotpress' ),
-							'waiting'                                     => __( 'Waiting approval', 'glotpress' ),
-							'rejected'                                    => __( 'Rejected', 'glotpress' ),
-							'untranslated'                                => __( 'Without current translation', 'glotpress' ),
-							'either'                                      => __( 'Any', 'glotpress' ),
+							'current'      => __( 'Current only', 'glotpress' ),
+							'old'          => __( 'Approved, but obsoleted by another translation', 'glotpress' ),
+							'waiting'      => __( 'Waiting approval', 'glotpress' ),
+							'rejected'     => __( 'Rejected', 'glotpress' ),
+							'untranslated' => __( 'Without current translation', 'glotpress' ),
+							'either'       => __( 'Any', 'glotpress' ),
```

Comment:
I'm assuming that this is because of https://github.com/WordPress/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#array-alignment-maximum-column. What about increasing the number so it can be aligned with `current_or_waiting_or_fuzzy_or_untranslated`?

Reply:
I agree, I'm fixing the alignment.
What ammount do you suggest set it to?

Reply:
maxColumn 80 seems to be enough